### PR TITLE
Fix Styled Components 5 + Upgrade TS

### PR DESCRIPTION
**Pull request by CompuIves**

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4210/commits/041b2a768e30e7371f87e9f027f9b70d2a5ac3d8 failed.

**Comment by siddharthkp** [comment]
@CompuIves This [breaks](https://www.chromatic.com/snapshot?appId=5d6e8335ddc81f0020dbed33&id=5ecd2296867a5100220e3e07) checkbox component 🤔

File: package.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
     "lint-staged": "^9.2.5",
     "prettier": "1.19.1",
     "pretty-quick": "^2.0.1",
-    "typescript": "3.7.4",
+    "typescript": "3.9.2",
```
Comment (MichaelDeBoey) [inline package.json:97]:
```suggestion
    "typescript": "3.9.3",
```

File: packages/app/package.json [hunk 2]
```diff
@@ -343,7 +343,7 @@
     "terser": "^4.1.4",
     "terser-webpack-plugin": "^1.4.1",
     "thread-loader": "^2.1.2",
-    "typescript": "3.7.4",
+    "typescript": "3.9.2",
```
Comment (MichaelDeBoey) [inline packages/app/package.json:346]:
```suggestion
    "typescript": "3.9.3",
```

File: packages/common/package.json [hunk 3]
```diff
@@ -111,10 +111,10 @@
     "jest": "^24.9.0",
     "jest-environment-enzyme": "^7.1.0",
     "jest-enzyme": "^7.1.0",
-    "jest-styled-components": "^6.3.4",
+    "jest-styled-components": "^7.0.2",
     "jest-svg-transformer": "^1.0.0",
     "rimraf": "^2.6.3",
-    "typescript": "3.7.4",
+    "typescript": "3.9.2",
```
Comment (MichaelDeBoey) [inline packages/common/package.json:117]:
```suggestion
    "typescript": "3.9.3",
```

**Pull request merged**